pyproject.toml

```diff
@@ -26,6 +26,10 @@ django-admin-autocomplete-list-filter = "1.0.1"
 django-guardian = "2.4.0"
 humanfriendly = "^10.0"
 jsonfield = "3.1.0"
+xlib = "^0.21"
+pyautogui = "^0.9.54"
+openai = "^1.8.0"
+python-dotenv = "^1.0.1"
 
```

Comment:
same here — unintentionally checked in?

Reply:
my bad, poetry files were intentionally checked in since I added new packages for the start of the general vision model, however I forgot to appropriately add them in the requirements.txt for pip